hack/generate-manifest.sh

```diff
@@ -272,7 +291,12 @@ if $KIND; then
     # this adds a small delay before running the antrea-agent process, to give the antrea-ovs
     # container enough time to set up the br-phy bridge.
     # workaround for https://github.com/vmware-tanzu/antrea/issues/801
-    $KUSTOMIZE edit add patch startAgent.yml
+    if $COVERAGE; then
+        cp ../../patches/coverage/startAgentCov.yml .
+        $KUSTOMIZE edit add patch startAgentCov.yml
```

Comment:
I am confused: it seems that you already applied this patch earlier?
```go
if $COVERAGE; then
    mkdir coverage && cd coverage
    cp ../../patches/coverage/*.yml .
    touch kustomization.yml
    $KUSTOMIZE edit add base $BASE
    # this runs antrea-controller via the instrumented binary.
    $KUSTOMIZE edit add patch startControllerCov.yml
    # this runs antrea-agent via the instrumented binary.
    $KUSTOMIZE edit add patch startAgentCov.yml
    BASE=../coverage
    cd ..
fi 
```

Reply:
when using kind, it will update antrea-agent again, so here we need to check if --coverage is passed.